src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Assets.json

```diff
@@ -0,0 +1,51 @@
+[
+  {
+    "generateSourceMaps": false,
+    "inputs": [
+      "wwwroot/node_modules/graphiql/graphiql.css"
+    ],
+    "output": "wwwroot/Styles/graphiql.css"
+  },
+  {
+    "generateSourceMaps": false,
+    "inputs": [
+      "wwwroot/node_modules/graphiql/graphiql.js"
+    ],
+    "output": "wwwroot/Scripts/graphiql.js"
```

Comment:
The minification of graphiql.js is breaking the graphql explorer page when built for production. https://github.com/graphql/graphql-js/issues/1182
We can't merge until this is resolved.

Reply:
@Skrypt

Reply:
Have you did npm install first on that wwwroot folder? I'm taking a look at this right now.

Reply:
Alright I can repro. It's only the graphql.min.js file that is not working. For some reason the minification must not do it's job properly. I'm going to try different things.

Reply:
fixed

Reply:
Thanks for the speedy fix @Skrypt